### PR TITLE
Add temp R data files to .gitignore

### PR DESCRIPTION
**Pull request by jbpoline**
temporary R data files in .gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -75,6 +75,10 @@ Thumbs.db
 doc/api/generated
 doc/build
 doc/manual
+doc/notebooks/.ipynb_checkpoints
```
Comment (matthew-brett) [inline .gitignore]: How about adding this at top level to ignore this directory at any level - as in:

```
.ipynb_checkpoints/
```

File: .gitignore [hunk 2]
```diff
@@ -75,6 +75,10 @@ Thumbs.db
...
+doc/notebooks/NewsGroup.RData
```
Comment (matthew-brett) [inline .gitignore]: Put `.gitignore` file in `doc/notebooks` with these entries?

File: .gitignore [hunk 3]
```diff
@@ -75,6 +75,7 @@ Thumbs.db
 doc/api/generated
 doc/build
 doc/manual
+.ipynb_checkpoints
```
Comment (matthew-brett) [inline .gitignore]: Sorry to be boring - but would you mind adding the slash at the end to indicate this a directory with all contents ignored?

**Comment by jbpoline** [comment]
done ! 
but  do you know why 
doc/api/generated
doc/build
doc/manual
don't have the trailing / ?

**Comment by matthew-brett** [comment]
Dunno - but they should.

**Comment by matthew-brett** [comment]
Thanks for weathering my pedantry.

**Comment by jbpoline** [comment]
not a pb at all - I should have put the / on the other ones ...

**Pull request merged**